### PR TITLE
Merge release/v0.1.3 into master

### PR DESCRIPTION
**Pull request by thedadams**
A release branch was created for v0.1.3, and those changes need to be merged into master so the changes are included in future releases.

### CODE REVIEW

**Comment by kinarashah** [comment]
@thedadams thanks for opening this PR!! we added the change and then reverted it, so wasn't sure if we needed it but makes sense to add it for consistency...where's the last commit coming from (`Merge release/v0.1.3`)?

**Comment by StrongMonkey** [comment]
Looks good to me but can we do cherry-pick so that commits look cleaner?

**Comment by thedadams** [comment]
Cherry-picked the 3 commits.

**Review by StrongMonkey — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -6,13 +6,12 @@ replace k8s.io/client-go => k8s.io/client-go v0.20.2
 
 require (
 	github.com/go-logr/logr v0.3.0
-	github.com/rancher/eks-operator v1.0.6-rc1
-	github.com/rancher/fleet/pkg/apis v0.0.0-20210203165831-44af1553b47e
+	github.com/rancher/fleet/pkg/apis v0.0.0-20210225010648-40ee92df4aea
```
Comment (kinarashah) [inline go.mod:9]: qq: eks-operator got removed, is this expected? I still see it in master latest commit

Comment (thedadams) [reply go.mod:9]: I had to run `go mod tidy` after I fixed the merge conflicts and it was removed.

**Review by kinarashah — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (cbron) [reply go.mod:9]: Does master branch of operator already avoid the cluster-overwrite problem ?

Comment (thedadams) [reply go.mod:10]: Darren changed this part of it, so I would guess that it wouldn't have this problem anymore. Regardless, Daishan's PR should also fix the overwrite problem: https://github.com/rancher/rancher-operator/pull/34

**Review by cbron — APPROVED** [review]

**Pull request merged**